crmsh/bootstrap.py

```diff
@@ -143,7 +143,6 @@ def __init__(self):
                 COROSYNC_AUTH, "/var/lib/heartbeat/crm/*", "/var/lib/pacemaker/cib/*",
                 "/var/lib/corosync/*", "/var/lib/pacemaker/pengine/*", PCMK_REMOTE_AUTH,
                 "/var/lib/csync2/*", "~/.config/crm/*"]
-        self.use_ssh_agent = False
```

Comment:
Do not abuse dynamic language features. Adding fields dynamically make code unreadable.

Reply:
Some `crm cluster` subcommands don't have `--use-ssh-agent`, like `crm cluster remove`
I made this change to distinguish if the user used this option or not.

Reply:
Or, we can add a flag to mark which option is from argparse while calling
```
@classmethod
    def set_context(cls, options):
        ctx = cls()
        for opt in vars(options):
            setattr(ctx, opt, getattr(options, opt))
        ctx.initialize_user()
        return ctx

```

Reply:
`None` is enough for marking a field as not set.